doc/.readthedocs.yml

```
@@ -10,4 +10,7 @@ sphinx:
 
 python:
   install:
+    # Install doc-building requirements (sphinx, etc.)
     - requirements: docs/requirements.txt
+    # Install hypnotoad
+    - path: package
```

I _think_ this needs to be

```suggestion
    - path: .
    - method: pip
```

---

It's being very annoying. One thread said that the default if you include `-path:` in the install section is to use pip. That didn't seem to work, which was why I tried 'package'. Current guess is that putting two things in the same `install:` section didn't work (although I think that's what the example in the RTD docs said...).